core/src/main/scala/org/apache/spark/ui/jobs/StagePage.scala

```diff
@@ -848,7 +848,7 @@ private[ui] class TaskPagedTable(
         </div>
       </td>
       <td>{UIUtils.formatDate(task.launchTime)}</td>
-      <td>{formatDuration(task.taskMetrics.map(_.executorRunTime))}</td>
```

Comment:
Hi @MrDLontheway , please note that this change is intentionally introduced in #23081.